mgmtsystem_action/views/mgmtsystem_action.xml

```diff
@@ -0,0 +1,194 @@
+<odoo>
+    <data>
+
+        <record id="view_mgmtsystem_action_tree" model="ir.ui.view">
+            <field name="name">mgmtsystem.action.tree</field>
+            <field name="model">mgmtsystem.action</field>
+            <field name="type">tree</field>
+            <field name="arch" type="xml">
+                <tree string="Action">
+                    <field name="reference"/>
+                    <field name="name" string="Subject"/>
+                    <field name="type_action"/>
+                    <field name="user_id"/>
+                    <field name="date_deadline"/>
+                    <field name="stage_id"/>
+                    <field name="company_id" groups="base.group_multi_company"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="view_mgmtsystem_action_filter" model="ir.ui.view">
+            <field name="name">mgmtsystem.action.filter</field>
+            <field name="model">mgmtsystem.action</field>
+            <field name="type">search</field>
+            <field name="arch" type="xml">
+                <search string="Action">
+                    <group>
+                        <separator orientation="vertical"/>
+                        <field name="reference"/>
+                        <field name="date" select='1'/>
+                        <field name="user_id"/>
+                        <field name="system_id"/>
+                        <field name="company_id" groups="base.group_multi_company"/>
+                    </group>
+                    <newline/>
+                    <group expand="0" string="Group By...">
+                        <filter string="User" icon="terp-personal" domain="[]" context="{'group_by':'user_id'}"/>
+                        <separator orientation="vertical"/>
+                        <filter string="Type" icon="terp-personal" domain="[]" context="{'group_by':'type_action'}"/>
```

Comment:
Provide "name" to the filters, and that is the name to use in later `_search_default_` context references.